src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -237,10 +239,130 @@ private static void PromptRequiredProperties<T>(T manifest, VersionManifest vers
+
+                if (selectedItem == Resources.None_MenuItem)
+                {
+                    break;
+                }
+                else if (selectedItem == Resources.AllInstallers_MenuItem)
+                {
+                    Installer installerCopy = new Installer();
+                    PromptHelper.DisplayPropertiesAsMenuSelection(installerCopy, Resources.None_MenuItem);
+                    ApplyChangesToIndividualInstallers(installerCopy, installerManifest.Installers);
+                }
+                else if (selectedItem == Resources.DisplayPreview_MenuItem)
+                {
+                    Console.Clear();
+                    Console.WriteLine();
+                    Logger.InfoLocalized(nameof(Resources.DisplayPreviewOfSelectedInstaller_Message));
+                    var serializer = Serialization.CreateSerializer();
+                    string installerString = serializer.Serialize(installerManifest);
+                    Console.WriteLine(installerString);
+                    Console.WriteLine();
+                }
+                else
+                {
+                    Installer selectedInstaller = MatchMenuSelectionToInstaller(selectedItem, installerManifest);
+                    PromptHelper.DisplayPropertiesAsMenuSelection(selectedInstaller, Resources.None_MenuItem);
+                }
+            }
+        }
+
+        private static Installer MatchMenuSelectionToInstaller(string selectedItem, InstallerManifest installerManifest)
+        {
+            string[] selection = selectedItem.Split('|');
+            var matchedInstallers = installerManifest.Installers.Where(
+                item => item.Architecture == (InstallerArchitecture)Enum.Parse(typeof(InstallerArchitecture), selection[0]) &&
+                item.InstallerType == (InstallerType)Enum.Parse(typeof(InstallerType), selection[1], true) &&
+                item.InstallerUrl == selection[2]);
+
+            Installer selectedInstaller;
+            if (matchedInstallers.Count() > 1)
+            {
+                Scope scope;
+                Scope? nullableScope = null;
+                string installerLocale = string.Empty;
+
+                for (int i = 3; i < selection.Length; i++)
+                {
+                    // if parsing for scope enum fails, value must be an installer locale
+                    if (Enum.TryParse(selection[i], true, out scope))
+                    {
+                        nullableScope = scope;
+                    }
+                    else
+                    {
+                        installerLocale = selection[i];
+                    }
+                }
+
+                selectedInstaller = matchedInstallers.Single(
+                    item => nullableScope != null && item.Scope == nullableScope &&
+                    !string.IsNullOrEmpty(installerLocale) && item.InstallerLocale == installerLocale);

```

Comment:
won't this fail if scope has a value but locale doesn't?

Reply:
I changed the logic for this method around, this shouldn't be an issue anymore